### PR TITLE
feat(all_in_one): fuse app and push

### PR DESCRIPTION
**Pull request by TimothyZhang023**
make one apk for user

Signed-off-by: zts1993 <zts1993@gmail.com>

### CODE REVIEW

File: push/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -3,90 +3,104 @@
...
+            android:name="top.trumeet.mipushframework.event.RecentActivityActivity"
+            android:label="@string/recent_activity_title" />
+        <activity
+            android:name="top.trumeet.mipushframework.help.HelpActivity"
+            android:label="@string/helplib_title"
+            android:theme="@style/HelpTheme.DayNight" />
+
+        <meta-data
+            android:name="xposedmodule"
+            android:value="true" />
```
Comment (MlgmXyysd) [inline push/src/main/AndroidManifest.xml:83]: According to https://github.com/MiPushFramework/MiPushFramework/commit/9a1174c155281423e10e75c10d9d59e5fe84aacd , there should be no more xposed module related content in Push. At the same time, the xposed function in push has been moved to a new independent software [MiPushEnhancement](https://github.com/MiPushFramework/MiPushEnhancement), and these residual codes should be deleted.

Reference: https://t.me/MiPushFramework/12

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]

**Comment by Trumeet** [comment]
Ohh that's so many changes...

File: push/src/main/res/values-zh/strings.xml [hunk 2]
```diff
@@ -22,17 +22,16 @@
     <string name="xposed_description">小米推送机型伪装插件，如需使用请开启</string>
```
Comment (MlgmXyysd) [inline push/src/main/res/values-zh/strings.xml:22]: There are still some remnants since the last change.

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]

**Review by Trumeet — APPROVED** [review]
Seems good after some adjustments.

**Pull request merged**